slothy/targets/aarch64/cortex_a55.py

```diff
@@ -426,6 +428,7 @@ def get_min_max_objective(slothy):
     (vshl, vshl_d, vsshr, vushr, vuxtl): 1,
     (trn2, trn1, ASimdCompare): 1,
     (Ldr_Q): 2,
+    (AArch64NeonCount): 2,
```

Comment:
This is unfortunately not correct. SLOTHY measures the **inverse throughput PER execution unit** in cycles / instruction , while the SWOG contains the throughput in instructions / cycle. 

For the dform of this instruction it occupies 1 execution unit for 1 cycle, and, hence, the throughput should be 1. 
For the qform of this instruction it occupies 2 execution units for 1 cycle each, and hence, the throughput should be 1. 
--> irrespective of the q/d-form, the throughput is 1.